lib/AppInfo/Application.php

```diff
@@ -76,6 +76,9 @@ public function boot(IBootContext $context): void {
 					'traces_sample_rate' => $config->getSamplingRate(),
 					'profiles_sample_rate' => $config->getProfilesSamplingRate(),
 				]);
+
+				\OC_Util::addHeader('meta', ['name' => 'baggage', 'content' => \Sentry\getBaggage()]);
```

Comment:
private API :broken_heart:

Reply:
Yep didn't find a way through the public one, need to think a bit how that could be arranged

Reply:
Can we use a global middleware?

Reply:
We cannot access the template instance from the template response in the middleware, but I have a local draft for extending that API, will check the next days.

Reply:
Seems I won't find time, let me close this until it becomes more relevant again or anyone else wants to pick up